VideoAPI/VideoApi.AcceptanceTests/Features/InstantMessages.feature

```diff
@@ -27,3 +27,12 @@ And I have a remove messages from a conference request
 When I send the request to the endpoint
 Then the response should have the status NoContent and success status True
 And the chat messages are deleted
+
+@VIH-6021
+Scenario: Get chat messages for a participant
```

Comment:
Could you add a test to get im history for a participant who does not belong in the conference? Thanks

Reply:
added ac and int test for non existent participants